tests/e2e/gateway/03-assert.yaml

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
This file is deleted, is the assert of this file checked anywhere?

Reply:
Too bad i don't see it in the migrated test, i hope we fixed it in v0.1.4

Reply:
Fixed